tests/integration/component_setup.sh

```diff
@@ -39,11 +39,22 @@ setup_nodejs() (
   fi
 )
 
+# cd testcomponent-go && go build -o out && \
+#   pulumi plugin install resource testcomponent v0.1.0 --reinstall -f out && \
+#   cd .. && cd testcomponent2-go && go build -o out && \
+#   pulumi plugin install resource secondtestcomponent v0.1.0 --reinstall -f out && \
+#   cd .. && cd go && pulumi up || cd ..
```

Comment:
What are these comments for? Do they need to be kept?

Reply:
The comment is the one liner necessary to run the test manually. They can be removed.